### PR TITLE
fix(ssl): Use shiny new SSL certificate

### PR DESCRIPTION
**Pull request by jbuck**

### CODE REVIEW

File: aws/environments/EXAMPLE.yml [hunk 1]
```diff
@@ -3,17 +3,16 @@
...
 
-# the name of the ssl cert in IAM to use in the ELB
-# this must be compatible with your stack_name and hosted_zone
-# see http://docs.aws.amazon.com/IAM/latest/UserGuide/ManagingServerCerts.html
-ssl_certificate_name: exp20170412_wildcard_dev_lcip.org
-cdn_ssl_certificate_id: ASCAIAYDJPL5OT7LFEZSE
+# The ARN of the SSL certificate in ACM to use with Cloudfront & ELB
+# If you need to use us-west-2, use:
+# ssl_certificate_arn: arn:aws:acm:us-west-2:927034868273:certificate/ef6bb380-b02e-4433-9355-aec09dedd381
+ssl_certificate_arn: arn:aws:acm:us-east-1:927034868273:certificate/675e0ac8-23af-4153-8295-acb28ccc9f0f
```
Comment (jrgm) [inline aws/environments/EXAMPLE.yml]: Doesn't the Cloudfront ACM id have to be in us-east-1, even if your stack is built in us-west-2?

Comment (jrgm) [inline aws/environments/EXAMPLE.yml]: http://docs.aws.amazon.com/acm/latest/userguide/gs-cf.html
"3. ... To use an ACM certificate with CloudFront, you must request or import the certificate in the US East (N. Virginia) region."

Comment (jbuck) [inline aws/environments/EXAMPLE.yml]: Another good catch!

**Comment by jrgm** [comment]
LGTM, I think.

**Pull request merged**